list_test.go

```diff
@@ -15,7 +15,11 @@
 package hessian
 
 import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
Add new import clause for third-party library.

Reply:
ok